### DISCUSSION TITLE
Android (samsung s24 arm8) air not working, would like help

### DISCUSSION THREAD

**Discussion by 76playsred** [airsdk/Adobe-Runtime-Support#3411] [Q&A] [answered]
Whenever I open the android apk it just pulls up a screen with nothing to click and I can do anything and the air file I want to open won't let me because my phone doesn't recognize adobe air as a file opener
![Screenshot_20240818_015456](https://github.com/user-attachments/assets/db9058f2-b90e-4a5a-b69f-2008f4748bc7)

**Comment by ajwfrost** [comment]
It looks like you've put the "Runtime.apk" file onto your phone .. this isn't intended for direct installation now: it used to be that you could have this as a "shared runtime" and then your application could be installed onto the phone without needing its own "captive" runtime. But Adobe dropped support for that many years ago..

Instead, you need to use the ADT tool to package up an APK file. Similar arguments to how you created your .air file, but with "-target apk" after the "-package" instruction.

See https://airsdk.dev/docs/building/air-developer-tool for ADT info.. or let us know if you're using an IDE such as Adobe Animate and we can point to some relevant instructions..

thanks

**Reply by 76playsred** [reply]
Thank you for your answer and help but if I could one more question because I have no clue how to run adt because it is a jar file and I can't run it without the jave runtime environment which isn't available to be downloaded on Android.

**Reply by ajwfrost** [reply]
Yes, ADT is a tool to be run on a desktop, when you're creating your AIR application.

So can I check, regarding the application you're trying to get into the Android device - is this something you are creating? Or is this something you'd received just as an .air file?
- if you're creating it, then ADT is what's creating the .air file, so if you let us know how you're getting that, we can hopefully help in how to instead create an APK file for Android
- if you're getting it from a third party, please ask them to provide an installation package for Android, the .air file cannot be used for this.

thanks

**Reply by 76playsred** [reply]
It's a air file that I have gotten from another source and why I'm here so I'm trying to find a way to run the air file

**Reply by ajwfrost** [reply, marked as answer]
Ah okay. So, a .air file can be installed on a desktop Windows or macOS computer, if they have the "shared" AIR runtime installed (from https://airsdk.harman.com/runtime)

For mobile applications (and Linux), the AIR application needs to be packaged for the particular platform by the developer, so you'll need to contact them..

thanks

**Reply by 76playsred** [reply]
Thank you
